### PR TITLE
Styling table with amounts

### PR DESCRIPTION
**Pull request by josehernandezfintecheandomx**
## Description

Styling tables listing accounts in the amount attributes alignment to the right

## Screenshots, if any
- Client general account 
Screenshot 2022-12-15 at 9 17 08

- Loan account
Repayment Schedule
Screenshot 2022-12-15 at 9 10 48

Transactions
Screenshot 2022-12-15 at 9 11 04

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [ ] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by adamsaghy** [comment]
@josehernandezfintecheandomx Why to align to right? I looked better as it is now. Why is this needed?

**Comment by josehernandezfintecheandomx** [comment]
> @josehernandezfintecheandomx Why to align to right? It looked better as it is now. Why is this needed?

In some cases without alignment when you have many data the values can not read in a easy way, plus they have not number format with thousands indicator

File: src/app/savings/savings-account-view/transactions-tab/transactions-tab.component.html [hunk 1]
```diff
@@ -20,33 +20,33 @@ <h3>All Transactions</h3>
...
-        <td mat-cell *matCellDef="let transaction"> {{ isDebit(transaction.transactionType) ? transaction.amount : 'N/A'}} </td>
+        <th mat-header-cell class="center" *matHeaderCellDef> Debit </th>
+        <td mat-cell class="r-amount" *matCellDef="let transaction"> {{ isDebit(transaction.transactionType) ? (transaction.amount | number) : '-.-'}} </td>
       </ng-container>
 
       <ng-container matColumnDef="credit">
-        <th mat-header-cell *matHeaderCellDef> Credit </th>
-        <td mat-cell *matCellDef="let transaction"> {{ !isDebit(transaction.transactionType) ? transaction.amount : 'N/A' }} </td>
+        <th mat-header-cell class="center" *matHeaderCellDef> Credit </th>
+        <td mat-cell class="r-amount" *matCellDef="let transaction"> {{ !isDebit(transaction.transactionType) ? (transaction.amount | number) : '-.-' }} </td>
```
Comment (adamsaghy) [inline src/app/savings/savings-account-view/transactions-tab/transactions-tab.component.html:39]: Why "-.-" better than N/A?

Comment (adamsaghy) [reply src/app/savings/savings-account-view/transactions-tab/transactions-tab.component.html:39]: Also nowhere all the other balance has no fallback value. Why to add to debit and credit?

**Comment by adamsaghy** [comment]
> > @josehernandezfintecheandomx Why to align to right? It looked better as it is now. Why is this needed?
> 
> @adamsaghy In some cases without alignment when you have many data the values can not read in a easy way, plus they have not number format with thousands indicator

I am okay with the number format, but if you take a look on the screenshot you have just attached, it looks weird. My recommendation is to align the header title and the values to the same side. Either all of the aligned to left, or center or right. @josehernandezfintecheandomx What do you think?

File: src/app/savings/savings-account-view/transactions-tab/transactions-tab.component.html (hunk 1 above)
Comment (josehernandezfintecheandomx) [reply src/app/savings/savings-account-view/transactions-tab/transactions-tab.component.html]: Done, resolved

**Pull request merged**